### PR TITLE
[GDI32_APITEST] Add Tahoma font to GetTextMetrics apitest set

### PR DESCRIPTION
**Pull request by Doug-Lyons**
## Add Tahoma font to GetTextMetrics apitest set.

_Tahoma is available in Windows and ReactOS both, so this allows us to compare these fonts in both systems._

### CODE REVIEW

**Comment by KyleKatarn31** [comment]
Please keep in mind that some glyphs are mising in ROS's Tahoma compared to Windows
image

But your PR is interesting in particular with regard to the recent issues faced here : https://github.com/reactos/reactos/pull/2607

File: modules/rostests/apitests/gdi32/GetTextMetrics.c [hunk 1]
```diff
@@ -464,6 +517,8 @@ START_TEST(GetTextMetrics)
         {
             const TEST_ENTRY *test = &font->tests[k];
 
+            printf("Testing '%s'.\n", font->entry_name);
```
Comment (learn-more) [inline modules/rostests/apitests/gdi32/GetTextMetrics.c:520]: please use `trace` here, which is a builtin winetest function (and can be configured with test parameters)

**Comment by learn-more** [comment]
https://testbot.winehq.org/JobDetails.pl?Key=70155

File: modules/rostests/apitests/gdi32/GetTextMetrics.c (hunk 1 above)
Comment (Doug-Lyons) [reply modules/rostests/apitests/gdi32/GetTextMetrics.c]: Done and moved to a better location as well. Thanks.

**Comment by Doug-Lyons** [comment]
Thanks for the commit, @katahiromz.

**Pull request merged**